fiftyone/factory/repos/delegated_operation_doc.py

```
@@ -72,6 +73,7 @@ def from_pymongo(self, doc: dict):
         self.pinned = doc.get("pinned", None)
         self.dataset_id = doc.get("dataset_id", None)
         self.run_link = doc.get("run_link", None)
+        self.log_status = doc.get("log_status", None)
```

Ruff (0.8.2)

Use `doc.get("log_status")` instead of `doc.get("log_status", None)`

Replace `doc.get("log_status", None)` with `doc.get("log_status")`

(SIM910)